pkg/transport/http/options.go

```diff
@@ -24,6 +24,10 @@ const (
 	MethodOption = "method"
 	// CodecOption represents the HTTP listener codec option key
 	CodecOption = "codec"
```

Comment:
is it still used or should be removed?

Reply:
This could still be used when wanting to simply define a codec. The codec options set both the request and response codec and the `request_codec` and `response_codec` could be used to define different codecs for both the request and response.